test-harness/tests/conftest.py

```diff
@@ -7,10 +7,10 @@
 import pytest
 
 from runner.common import (
-    DNSMock,
```

Comment:
Could you also please clean up the DNSMock class and all stuff related to dnsmasq from `docker/` container?